### ISSUE TITLE
Many answers missing from the algorithms section

### ISSUE THREAD

**Issue by 0xpurpl3** [swapnanildutta/Hackerrank-Codes#100]
Hey there, looks like that there are not many answers in the 'Algorithm' Section of the Repo. I have solved many of the problems and i would like to contribute to this project. May I?

btw here's my hackerrank profile link: https://www.hackerrank.com/0xpurpl3

**Comment by swapnanildutta** [comment]
Yeah Sure. You're most welcome. Just add the HackerRank Question on top and then the code. And also Update the README in the directory.

**Comment by 0xpurpl3** [comment]
sorry i didn't understand. What exactly am i supposed to update in the README file?

**Comment by swapnanildutta** [comment]
The description of the directory, how to execute the codes there and which codes are present there.

**Comment by 0xpurpl3** [comment]
ooh okay! . Will do that

**Comment by rgarg-2711** [comment]
want to add some problems
